src/equations/shallow_water_2d.jl

```diff
@@ -364,6 +458,34 @@ Further details are available in Theorem 1 of the paper:
   shallow water equations on unstructured curvilinear meshes with discontinuous bathymetry
   [DOI: 10.1016/j.jcp.2017.03.036](https://doi.org/10.1016/j.jcp.2017.03.036)
 """
+@inline function flux_wintermeyer_etal(u_ll, u_rr, orientation::Integer, equations::ShallowWaterEquations2D)
```

Comment:
The argument in the docstring above  should become `orientation_or_normal_direction`

Reply:
Done